cmplibrary/src/main/java/com/sourcepoint/gdpr_cmplibrary/ConsentAction.java

```diff
@@ -16,4 +16,13 @@
         this.requestFromPm = requestFromPm;
         this.pmSaveAndExitVariables = pmSaveAndExitVariables;
     }
+
+    public static ConsentAction getEmptyDismissAction(boolean isPmOn){
```

Comment:
I think it's cleaner with a ternary comparison for this case:
  ```JAVA
return isPmOn ? new ConsentAction(2, "", true, new JSONObject()) : new ConsentAction(15, "", false, new JSONObject());
```
It's just a matter of style 😅 What do you think?

Reply:
Yeah , Good suggestion